packages/worker/src/api/controllers/global/configs.ts

```diff
@@ -304,6 +333,30 @@ function enrichOIDCLogos(oidcLogos: OIDCLogosConfig) {
   )
 }
 
+async function enrichAIConfig(aiConfig: AIConfig) {
+  // Strip out the API Keys from the response so they don't show in the UI
+  for (let key in aiConfig.config) {
+    if (aiConfig.config[key].apiKey) {
+      aiConfig.config[key].apiKey = PASSWORD_REPLACEMENT
+    }
+  }
+
+  // Return the Budibase AI data source as part of the response if licensing allows
+  const budibaseAIEnabled = await pro.features.isBudibaseAIEnabled()
+  const defaultConfigExists = Object.keys(aiConfig.config).some(key => aiConfig.config[key].isDefault)
+  if (budibaseAIEnabled) {
+    aiConfig.config["budibase_ai"] = {
+      provider: "OpenAI",
+      active: true,
+      isDefault: !defaultConfigExists,
+      defaultModel: env.BUDIBASE_AI_DEFAULT_MODEL,
```

Comment:
Do we want some sort of validation in this env var in case someone changes it to something invalid?

Reply:
I think it's ok, as it will be us who sets it and it can only be from the list of GPT models, so fairly low risk